sdk/config_helpers.go

```diff
@@ -624,9 +626,10 @@ func AddAuxfileToNginxConfig(
 }
 
 const (
-	plusAPIDirective = "api"
-	ossAPIDirective  = "stub_status"
-	apiFormat        = "http://%s%s"
+	plusAPIDirective          = "api"
+	ossAPIDirective           = "stub_status"
+	apiFormat                 = "http://%s%s"
+	predefinedAccessLogFormat = "$remote_addr - $remote_user [$time_local] \"$reques\" $status $body_bytes_sent \"$http_referer\" \"$http_user_agent\""
```

Comment:
Should be `$request` instead of `$reques`

Reply:
@dhurley thanks catching.
I fixed.
https://github.com/nginx/agent/commit/b57985802f997447ebf7d433494b3c86e98ca01f